tutorials/arvo/claz.md

```diff
@@ -0,0 +1,76 @@
++++
+title = "claz"
+weight = 4
+template = "doc.html"
++++
+
+This document outlines some advanced techniques for interacting with Azimuth contracts from Urbit itself. It's strongly discouraged to do this unless you're very confident as Azimuth transactions will cost Etherium to complete. However, until you upload a signed transaction to the block chain no transaction will take place so it's safe to explore these tools. It will also be very helpful to have at least a rudimentary understanding of Hoon in order to read and modify parts of the code.
+
+First you will want to boot a fakezod. You can do this with your live running ship but at this time this will need you to modify the code so it's best to do this on a ship you can make mistakes on. If you don't know how to boot a fakezod see this guide for instructions.
+
+Be sure to have mounted your home desk so you can easily edit files on it.
+The first edit that will probably need to be made is in [`app/claz.hoon`](https://github.com/urbit/urbit/blob/master/pkg/arvo/app/claz.hoon#L14). The listed url will need to be changed to 
+
+https://mainnet.infura.io/v3/2599df54929b47099bda360958d75aaf
```

Comment:
Most of our endpoints aren't super secret, but it feels wrong to just up and share them like this. Besides, if you can't obtain an endpoint of your own (Infura is free for small-scale use!) then this guide is not for you.